test/Groups/conjugation.jl

```diff
@@ -116,41 +116,43 @@
 
 end
 
-function TestConjCentr(G,x)
-   Cx = centralizer(G,x)[1]
-   cc = conjugacy_class(G,x)
-   @test index(G,Cx)==length(cc)
-   T=right_transversal(G,Cx)
-   @testset for y in cc
-       @test count(t -> y==x^t, T) == 1
+let
```

Comment:
Is this really necessary? I thought `@testset` starts a new scope and so these functions stay "local" inside that scope anyway. A quick test seems to confirm that:
```
julia> using Test

julia> @testset "bla" begin
         foo(x) = x + 1
       end
Test Summary: |Time
bla           | None  0.0s
Test.DefaultTestSet("bla", Any[], 0, false, false, true, 1.695032056975692e9, 1.695032056992175e9, false)

julia> foo(1)
ERROR: UndefVarError: `foo` not defined
Stacktrace:
 [1] top-level scope
   @ REPL[3]:1
```